sa/sa.go

```diff
@@ -1277,6 +1277,68 @@ func (ssa *SQLStorageAuthority) FQDNSetExists(ctx context.Context, names []strin
 	return count > 0, err
 }
 
+// PreviousCertificateExists returns true iff there was at least one certificate
+// issued with the provided domain name, and the most recent such certificate
+// was issued by the provided registration ID.
+func (ssa *SQLStorageAuthority) PreviousCertificateExists(
+	ctx context.Context,
+	req *sapb.PreviousCertificateExistsRequest,
+) (*sapb.Exists, error) {
+	t := true
+	exists := &sapb.Exists{Exists: &t}
+
+	f := false
+	notExists := &sapb.Exists{Exists: &f}
+
+	// Find the most recently issued certificate containing this domain name.
+	var serials []string
+	_, err := ssa.dbMap.Select(
+		&serials,
+		`SELECT serial FROM issuedNames
+		WHERE reversedName = ?
+		ORDER BY notBefore DESC
+		LIMIT 1`,
```

Comment:
We talked about this OOB - Can you leave a comment in this function somewhere that explains we understand this "query 1 most recent serial for name, then check reg ID" behaviour won't allow reuse for cases where a name is issued by multiple account IDs and are accepting that limitation with the intent to consider further iteration down the road if required?